client/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,118 @@
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import * as Yup from 'yup';
+import login from '../../helpers/APICalls/login';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import Image from '/Users/decagon/team-candy-cane/client/src/Images/doglanding.png';
+
+export default function LandingPage(): JSX.Element {
+  const classes = useStyles();
+  const { updateLoginContext } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const handleSubmit = (
+    { place, drop }: { place: string; drop: string },
+    { setSubmitting }: FormikHelpers<{ place: string; drop: string }>,
+  ) => {
+    login(place, drop).then((data) => {
+      if (data.error) {
+        setSubmitting(false);
+        updateSnackBarMessage(data.error.message);
+      } else if (data.success) {
+        updateLoginContext(data.success);
+      } else {
+        // should not get here from backend but this catch is for an unknown issue
+        console.error({ data });
+
+        setSubmitting(false);
+        updateSnackBarMessage('An unexpected error occurred. Please try again');
+      }
+    });
+  };
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
+
+      <Grid item className={classes.root2}>
+        <Typography className={classes.welcome} component="h1" variant="h5">
+          Find the care your dog deserves
+        </Typography>
+        <Formik
+          initialValues={{
+            place: '',
+            drop: '',
+          }}
+          validationSchema={Yup.object().shape({
+            place: Yup.string(),
+            drop: Yup.string(),
+          })}
+          onSubmit={handleSubmit}
+        >
+          {({ handleSubmit, handleChange, values, touched, errors }) => (
+            <form onSubmit={handleSubmit} className={classes.form} noValidate>
+              {<Typography className={classes.label}>Where</Typography>}
+              <TextField
+                id="place"
+                fullWidth
+                margin="normal"
+                label=""
+                placeholder="AnyWhere"
+                InputLabelProps={{
+                  shrink: true,
+                }}
+                InputProps={{
+                  classes: { input: classes.inputs },
+                }}
+                variant="outlined"
+                name="place"
+                autoComplete="place"
+                autoFocus
+                helperText={touched.place ? errors.place : ''}
+                error={touched.place && Boolean(errors.place)}
+                value={values.place}
+                onChange={handleChange}
+              />
+              {<Typography className={classes.label}>Drop IN /Drop OFF</Typography>}
```

Comment:
Please avoid using CAPITALCASE for text, some screen readers interpret this text as abbreviations, you can resort to using a CSS class to transform text instead.

Reply:
ok